### ISSUE TITLE
path too long - "The system cannot find the specified path"

### ISSUE THREAD

**Issue by phillipmohr** [dockware/dockware#56]
Hi. I'm trying to install Shopware 6 locally. I'm following this documentation: https://docs.dockware.io/development/start-developing

I get an error at step 2, the 2nd line `docker cp shopware:/var/www/html/. ./src`

The error is: open C:\r\src\vendor\shopware\administration\Resources\app\administration\src\module\sw-cms\elements\product-listing\config\components\sw-cms-el-config-product-listing-config-filter-properties-grid\sw-cms-el-config-product-listing-config-filter-properties-grid.html.twig: The system cannot find the specified path.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Image: dockware/dev:latest
 - Docker: Docker for Windows 3.3.3

**What I tried to get rid of the error**
- made the path as short as possible (project is under C:\ + project dir name is just "r"
- Activated long paths in windows (see screenshot attached)
![windows-group-pol](https://user-images.githubusercontent.com/30236221/119231104-69e61a00-bb1f-11eb-9efe-0300007702bf.PNG)

Feel free to request additional information if needed. 

Thanks for help!

**Comment by phillipmohr** [comment]
I found a workaround and got rid of the error:

```
docker exec -it shopware bash
cd ..
zip html html -r
```

exit container

```
cp shopware:/var/www/html.zip ./src
cd src 
unzip html.zip
```

This works, but unfortunately the watcher isn't.

I tried make watch-storefront and it compiled and everything, but it says the following in the console:
```
Failed to load resource: the server responded with a status of 404 (Not Found)
vendor-node.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
runtime.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
app.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
storefront.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
vendor-shared.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
runtime.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
app.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
storefront.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
```
![Uploading watcher-doesnt-work.PNG…]()

**Comment by boxblinkracer** [comment]
Hi

alright, so i had a boostday workshop today,
it basically works. you need to use http://localhost:9998
then it lazy-loads the storefront

but the changes from a plugin are not recognized though.
i found out, if i use the default ./bin/build-storefront.sh and THEN start the watcher with the makefile, ...then it works correctly.
so theres something that is missing in our makefile, maybe a wrong theme dump plugin...it might need to be done before the other command

but overall..it worked great for me then

**Comment by phillipmohr** [comment]
@boxblinkracer Thank you very much. I will try that out, unfortunately I now have another issue with Windows long paths [which I created here, too](https://github.com/dockware/dockware/issues/57). #57

**Comment by boxblinkracer** [comment]
Hi

one thing i found out during boostday was that custom files do olny work once build-storefront is done manually, and afterwards run the make watch-storefront, but then it worked for me

so try
* running dockware as usual + ports 9999:9999 + 9998:9998
* run build-storefront.sh inside docker
* run make watch-storefront inside docker

then http://localhost:9998 works for me

if its still not working, maybe we can arrange a short zoom...could you write me in slack (shopware channel) in that case?

**Comment by aloiki** [comment]
Hi, 

as described above the theme is not working after compiling. Tried it with the following commands:

run build-storefront.sh inside docker
run make watch-storefront inside docker

as you can see in the screenshots the assets are not found after updating a theme.
<img width="1436" alt="Bildschirmfoto 2022-08-03 um 15 12 50" src="https://user-images.githubusercontent.com/41948154/182700734-025502b2-35b7-4ac8-a271-bca5c1a33b34.png">

**Comment by boxblinkracer** [comment]
Hi

what URL are you using? do you use the one with port 9998? if you just use localhost then it looks like your screenshot

**Comment by boxblinkracer** [comment]
hi @aloiki 
sry, i didnt see the "as described above"...it was already night haha
what dockware image and tag are you using? i would check it out
thx

**Comment by floosenlord** [comment]
@phillipmohr @aloiki 
I've had a similar issue today.
Don't know if you are still having this issue, but leaving some info for other people stumbling upon this thread in the future:

After using `make watch-storefront` the frontend was displayed and hot-loading correctly using port 9998.
Afterwards I wanted to compile my theme changes, so I stopped using the watcher and executed `bin/console theme:compile`.
I had the same error as @phillipmohr, specifically the compiled JS files returning a 404.

In the makefile I found a section to stop the storefront watch mode:
```
stop-watch-storefront: ## Reverts everything back to normal operation
        @rm -rf /var/www/html/.htaccess.watch
```
After calling `make stop-watch-storefront` using the regular storefront (just `localhost` without a port) was working again.
